operations/phlare/helm/phlare/values.yaml

```
@@ -66,7 +66,12 @@ phlare:
     # We usually recommend not to specify default resources and to leave this as a conscious
     # choice for the user. This also increases chances charts run on environments with little
     # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # lines, adjust them as necessary, and remove the curly braces after 'resources'.
+    #
+    # Note that if memory consumption is higher than you would like, you can decrease the interval
+    # that profiles are written into blocks by setting `-phlaredb.max-block-duration` in the `extraArgs`
```

I think a full example yaml would be even more helpful. What do you think if do it like this?
```suggestion
    # that profiles are written into blocks by setting `-phlaredb.max-block-duration` in the `extraArgs`
    
    extraArgs:
      phlaredb.max-block-duration: 30m
```